recipe/meta.yaml

```diff
@@ -32,8 +32,8 @@ test:
   commands:
     - test -f ${PREFIX}/share/cmake/pybind11/pybind11Config.cmake                                     # [unix]
     - if exist %LIBRARY_PREFIX%\share\cmake\pybind11\pybind11Config.cmake (exit 0) else (exit 1)      # [win]
-    - test $(python -c "import pybind11 as py; print(py.get_include())") == "${PREFIX}/include"             # [unix]
-    - test $(python -c "import pybind11 as py; print(py.get_include())") == "%LIBRARY_PREFIX%\include"      # [win]
+    - test -f ${PREFIX}/include/pybind11/pybind11.h      # [unix]
+    - if exist %LIBRARY_INC%\pybind11\pybind11.h (exit 0) else (exit 1)      # [win]
```

Comment:
I would have liked if we kept the `py.get_include()` verification since that is the public API some users might be using, too.

Reply:
oh, still below. ok!